src/components/SearchPage/SearchInputComponent.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import TextField from '@material-ui/core/TextField';
+import SearchIcon from '@material-ui/icons/Search';
+import { IconButton, makeStyles } from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+import { useDebouncedCallback } from 'use-debounce/lib';
+import { searchQueryChangeAction } from './SearchPageAction';
+import { useEffect } from 'react';
+import { colors } from 'colors';
+import ClearIcon from '@material-ui/icons/Clear';
+
+const useStyles = makeStyles(() => ({
+  multilineColor: {
+    color: colors.iconColor,
+  },
+  customInput: {
+    color: colors.iconColor,
+    padding: '0px 10px',
+  },
+  list: {
+    width: '100%',
+    height: '100%',
+    paddingLeft: 16,
+    marginTop: -30,
+  },
+  listItem: {
+    padding: 0,
+    height: 64,
+  },
+  icon: {
+    paddingLeft: 12,
+    paddingRight: 12,
+  },
+  svgIcon: {
```

Comment:
What is the difference between `icon` and `svgIcon` ? 
Aren't all icons in svg format?

I think we can rename `icon` to `listItemIcon` and `svgIcon` to `icon`.

Reply:
deleted